modules/twinklewarn.js

```diff
@@ -1081,9 +1085,6 @@ Twinkle.warn.prev_article = null;
 Twinkle.warn.prev_reason = null;
 
 Twinkle.warn.callback.change_category = function twinklewarnCallbackChangeCategory(e) {
-	if (!Twinkle.getPref('oldSelect')) {
-		$('select[name=sub_group]').chosen('destroy');
```

Comment:
Do we really not need the `.destroy()` anymore?  I would've thought so, but seems select2 handles things much more smoothly!

Reply:
select2 is always two steps ahead of chosen :)